.env

```diff
@@ -0,0 +1,7 @@
+PORT=3000
+
+CHAIN_ID=1337
```

Comment:
I do not believe these will actually work without a prefix: https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
You're right, there were a few other issues I fixed as well.